external/radiation/tests/test_wrapper_api.py

```diff
@@ -115,25 +115,27 @@ def __init__(self):
 }
 
 
-def test_config_validation(regtest):
```

Comment:
This is mostly just better organization of tests that accomplish the same things